html/config.html

```diff
@@ -114,7 +114,87 @@ <h3>Thermal Motion<br></h3>
         <div class="container pt-1 pl-0">
             <button id="save-button" type="button" onclick="saveThermalMotionConfig()" class="btn btn-primary">Save ThermalMotion Config</button>
         </div>
+
+        <div class="container pt-5 pl-0">
+            <h3>Comms port<br></h3>
+        </div>
+
+        <div class="row">
+            <div class="col-xs-8">
+                <div class="table-responsive">
+                    <table class="table">
+                        <tbody>
+                            <tr>
+                                <td>Enable</td>
+                                <td><input type="checkbox" id="input-comms-enable"></td>
+                            </tr>
+                            <tr></tr>
+                                <td>Trap Default State</td>
+                                <td><input type="checkbox" id="input-comms-trap-default"></td>
+                            </tr>
+
+                            <tr>
+                                <td>Bluetooth Enablued</td>
```

Comment:
*<td>Bluetooth Enabled</td>
though kind of funny to leave as EnaBLUEd

Reply:
Haha, should probably fix it..